mobile/native/src/event/event_hub.rs

```diff
@@ -44,7 +44,7 @@ impl EventHub {
 
     /// Publishes the given event to all subscribers. Note, that this will be executed in a loop.
     pub fn publish(&self, event: &EventInternal) {
-        tracing::debug!("Publishing event {:?}", event);
+        tracing::debug!(event = %event, "Publishing an internal event");
```

Comment:
🤔 wouldn't `event = %event` be equivalent to just `%event`? I thought that the `a = %b` notation just renames b to a in tracing?

Reply:
Yes it's the same. Personally I prefer the explicit way of writing, i.e. `event = %event` over `%event`. I find it easy to grep over the code if I want to find log messages with `event` in it.

Reply:
I have no strong feelings, so if it works for you better, it's fine :) just wasn't sure whether you were aware of this!